src/pymorize/generic.py

```diff
@@ -252,3 +253,35 @@ def trigger_compute(data, rule_spec, *args, **kwargs):
         return data.compute()
     # Data doesn't have a compute method, do nothing
     return data
+
+
+def sort_dimensions(data, rule_spec):
+    """
+    Sorts the dimensions of a DataArray based on the array_order attribute of the
+    rule_spec. If the array_order attribute is not present, it is inferred from the
+    dimensions attribute of the data request variable.
+    """
+    missing_dims = rule_spec.get("sort_dimensions_missing_dims", "raise")
```

Comment:
I think we have a better way of doing this now, namely via `rule_spec._pymorize_cfg`. I think that kind of configuration is better placed there, but maybe this is something that should be specific to each rule separately, I am not sure...